src/DependenciesPackager/CrossgenContext.cs

```diff
@@ -192,22 +365,25 @@ private DirectoryInfo CreateSubDirectory(string cacheBasePath, PackageEntry pack
                 package.Library.Identity.Version.ToNormalizedString().ToLowerInvariant(),
                 Path.GetDirectoryName(asset.RelativePath));
 
-            _logger.LogInformation($"Creating sub directory on {subDirectoryPath}.");
-
             return Directory.CreateDirectory(subDirectoryPath);
         }
 
-        private string GetTempStagingFolder(string parent)
+        public void CreateResponseFile()
         {
-            var publishFolder = _context.GetStagingFolderPath(parent);
-
-            if (!Directory.Exists(publishFolder))
-            {
-                _logger.LogInformation($"Creating directory {publishFolder}.");
-                Directory.CreateDirectory(publishFolder);
-            }
+            _responseFilePath = Path.Combine(_destinationFolder, "aspnet-crossgen.rsp");
+            var builder = new StringBuilder();
+            builder.AppendLine($"/Platform_Assemblies_Paths \"{string.Join(";", _referenceAssemblyPaths)}\"");
```

Comment:
Actually, my concern was about argument escaping with spaces and backslashes. It is not just enough to surround arguments containing spaces with quotes. You also have to examine if an argument contains spaces and backlashes. See https://github.com/aspnet/DotNetTools/blob/e0f85971a32c21f4246b41ffabfca60e039b13f4/src/Shared/ArgumentEscaper.cs for example.